shell/src/test/java/alluxio/cli/fsadmin/command/ConfigurationCommandTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.command;
+
+import alluxio.cli.fsadmin.report.ConfigurationCommand;
+import alluxio.client.MetaMasterClient;
+import alluxio.wire.ConfigProperty;
+
+import org.hamcrest.collection.IsIterableContainingInOrder;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ConfigurationCommandTest {
+  private MetaMasterClient mMetaMasterClient;
+
+  @Before
+  public void prepareDependencies() throws IOException {
+    // Prepare mock meta master client
+    mMetaMasterClient = Mockito.mock(MetaMasterClient.class);
+    List<ConfigProperty> configList = prepareConfigList();
+    Mockito.when(mMetaMasterClient.getConfiguration())
+        .thenReturn(configList);
+  }
+
+  @Test
+  public void configuration() throws IOException {
+    try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+         PrintStream printStream = new PrintStream(outputStream, true, "utf-8")) {
+      ConfigurationCommand configurationCommand = new ConfigurationCommand(mMetaMasterClient,
+          printStream);
+      configurationCommand.run();
+      String output = new String(outputStream.toByteArray(), StandardCharsets.UTF_8);
+      List<String> expectedOutput = Arrays.asList("Alluxio configuration information: ",
+          "Property                                 Value                                     "
```

Comment:
This would read a lot better if we could put it on one line. We should be able to exclude it from checkstyle to allow this https://stackoverflow.com/questions/4023185/how-to-disable-a-particular-checkstyle-rule-for-a-particular-line-of-code

The Alluxio checkstyle file is in `build/checkstyle/alluxio_checks.xml`

Reply:
Changed in alluxio_checks.xml
and add 
 // CHECKSTYLE.OFF: LineLengthExceed - Much more readable
// CHECKSTYLE.ON: LineLengthExceed
in CapacityCommandTest and ConfigurationCommandTest to make expected output more readable. Thanks for the advice, later contributors can know what's report shell results better.